### DISCUSSION TITLE
Switch Action Status Veränderung des Levelwerts

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2203] [Q&A]
Hallo, 
ich bei ein Newbie ;-) und habe leider nicht gefunden wonach ich suche... 
Wie kann den der Status des Power on / off in Abhängigkeit des Levelwerts auf an bzw. aus geschaltet werden.
Ich versuche leider verzweifelt, dass sich der Power on/off - Schalter auf an bzw. aus stellt, wenn ich einen Dimmwert auswähle...

Wer kann mir weiterhelfen?
![Power schalten](https://user-images.githubusercontent.com/11274233/215143535-5fe346ab-2357-4cef-ad55-7d0fe16f788d.jpg)

Viele Grüße

Georg

**Comment by Zefau** [comment]
Das sollte automatisch passieren. Zeig mal deine Konfiguration.

**Comment by tschortschde** [comment]
Danke fürdie Rückmeldung
Das dachte ich auch ;-)
Meine Konfiguration:
![Licht schalten](https://user-images.githubusercontent.com/11274233/215268242-6c04f555-c5e5-4b93-b761-d44ee849d33e.jpg)
![Licht dimmen](https://user-images.githubusercontent.com/11274233/215268246-b95bb8a7-1295-4065-b1a5-ef2b41b9aa8c.jpg)
![Licht Dimmwert](https://user-images.githubusercontent.com/11274233/215268248-33ee5c57-61b6-4d7f-8c3b-ed6c0a698ccf.jpg)
![Widget](https://user-images.githubusercontent.com/11274233/215268254-5cb90df6-dec7-49f1-82d7-953c040e36d8.jpg)

Wo ist mein Fehler?
VG Georg

**Comment by Zefau** [comment]
Die Standard-Konfiguration greift nur für die Standard-Datenpunkt-Bezeichner. Also bei Licht sind die Standard-Datenpunkt-Bezeichner `level` / `dimmer` / `brightness`.

Alternativ kannst du deine Bezeichner beibehalten und manuell die Standard-Konfiguration setzen:
```
{
	"unit": val => val > 0 ? " %" : null,
	"stateStyle": {
		"default": {},
		"0": {
			"color": "#999"
		}
	},
	"icon": {
		"default": "lightbulb-on",
		"0": "lightbulb-off-outline"
	},
	"display": {
		"0": "aus"
	}
}
```

**Comment by tschortschde** [comment]
Habe nun die Standard-Datenpunkte verwendet, aber irgendwie funktioniert es nicht bzw. steh ich auf dem Schlauch ;-((( 
Obwohl ich den Regler "Level" noch unten auf "0" regle, schaltet der Schalter nicht auf aus...
Was nur mache ich falsch?
![Screenshot 2023-01-28 171213](https://user-images.githubusercontent.com/11274233/215277041-f4e30dd0-6c19-407d-9a4b-2da42c356ffe.jpg)

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/geraet/datenpunkt-eigenschaft/on-und-off-v3

**Comment by tschortschde** [comment]
Vielen Dank für die Unterstützung. 
nach dem Löschen der Datenpunkte und erneuten Einspielen des Knx -Projekts hat es dann funktioniert
